test/images/Makefile

```diff
@@ -0,0 +1,180 @@
+# is necessary; the underlying tools (podman and skopeo) may do that, but they
+# may still consume Dockerhub pull requests in the process.
+#
+# As with any Makefile, you can configure the execution by setting variables
+# on the make invokation:
+#
+#   make MAIN_REPO=quay.io/dhashimo docker.io/ssorj/quiver
+#
+# The command above will copy docker.io/ssorj/quiver into the quay repository
+# quay.io/dhashimo, instead of the default MAIN_REPO
+#
+#   make MAIN_REPO=quay.io/dhashimo everything
+#
+# Similarly, the command above will copy and build/generate everything, but
+# into the specified MAIN_REPO.
+#
+# The last operation is to copy images from MAIN_REPO to another repository,
+# which needs to be specified.
+#
+#   make COPY_REPO=192.168.0.1:5000/internal copy
+#
+# The command above will copy all images (external and generated, as listed
+# below) from the MAIN_REPO into the 'internal' repository within the
+# registry running at 192.168.0.1:5000.
+#
+# Individual items can also be specified.  For that, just add '_copy' to
+# their main target names:
+#
+#   make COPY_REPO=192.168.0.1:5000/internal nghttp2_copy
+#   make COPY_REPO=192.168.0.1:5000/internal docker.io/ssorj/quiver_copy
+#
+# This Makefile is intended to be executed manually, with the login of
+# individual users, and not by a robot account on a CI.
+#
+# ATTENTION: Remember that images copied for the first time into quay may
+#            be created as 'private' images, and need to be changed into
+#            'public' before they can be used by CI and other users.
+
+# --- configuration ---
+
+# tools
+SHELL = /bin/sh
+PODMAN = podman
+SKOPEO = skopeo
+
+# tool options
+
+# The options below create or transform the images into the docker format.
+# That is required for their use with Openshift 3.11
+FORMAT_OPTIONS = --format docker
+TRANSFORM_OPTIONS = --format v2s1
+
+# Repositories
+MAIN_REPO = quay.io/skupper
+COPY_REPO = localhost:5000/local
+
+# This is the list of external images that will be copied to $(MAIN_REPO).  If
+# the image specifies a tag, make sure to escape the colon with a backslash
+# (such as in docker.io/library/mongo\:5.0).  Otherwise, you'll get an error
+# like `multiple target patterns.  Stop.`
+EXTERNAL_IMAGES := \
+	docker.io/nginxinc/nginx-unprivileged \
+	docker.io/ssorj/quiver \
+	docker.io/library/postgres \
+	docker.io/library/redis \
+	docker.io/maistra/examples-bookinfo-productpage-v1\:0.12.0 \
+	docker.io/maistra/examples-bookinfo-ratings-v1\:0.12.0 \
+	docker.io/maistra/examples-bookinfo-details-v1\:0.12.0 \
+	docker.io/maistra/examples-bookinfo-reviews-v3\:0.12.0 \
+	docker.io/library/mongo\:5.0 \
+	docker.io/nginxinc/nginx-unprivileged\:stable-alpine
+
+# These are the images that are generated by this Makefile (as opposed to
+# created elsewhere and just copied here).  To add a new image, simply create
+# Dockerfile.IMAGENAME and add the IMAGENAME below.
```

Comment:
New images pushed into quay are private by default.
Maybe just add a note advising users to make image public.

Reply:
@fgiorgetti, I have something for that on line 66.  Do you think I should move or duplicate it here?